src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -406,8 +406,8 @@ private void assertAddressIsTransformed(Map<String, Object> address, Map<String,
     private void assertChildrenInfo(Map<String, Object> transformedCaseData) {
         List<Map<String, Object>> children = (List)(transformedCaseData.get("childrenInfo"));
 
-        assertChild(children.get(0), asList("Johny Bravo", "2000-03-12", "male", "son", "SON","New Zeeland"));
-        assertChild(children.get(1), asList("Anne Shirley", "1895-03-12", "female", "daughter", "Daughter","Canada"));
+        assertChild((Map)(children.get(0)).get("value"), asList("Johny Bravo", "2000-03-12", "male", "son", "SON","New Zeeland"));
```

Comment:
Another option would be to either have value as the first attribute of the ChildInfo class or to configure @JsonProperty("value.name"), for example (if that's possible).

Reply:
I'm working on ChildInfo model that would replace this Map-nightmare.